Security/src/AuthClient/Controllers/HomeController.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Steeltoe.Samples.AuthClient.Models;
+using System;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Steeltoe.Samples.AuthClient.Controllers;
+
+public sealed class HomeController(IHttpClientFactory clientFactory, ILogger<HomeController> logger) : Controller
+{
+    public IActionResult Index()
+    {
+        return View();
+    }
+
+    #region SSO
```

Comment:
Regions are considered an anti-pattern. Can we split this into multiple controllers?
If that isn't easy to do, consider putting them in separate files as partial classes, ie `HomeController.Sso.cs`, `HomeController.Mtls.cs`, etc.

Reply:
I meant to remove the regions entirely, they really aren't needed for such a small amount of code. We _could_ have separate controllers, but there's so little here I'm not convinced it's worth any additional effort

Reply:
I agree.